### PR TITLE
detail component

### PR DESCRIPTION
**Pull request by Akido123**
merged instructions and ingredients components into detail component

### CODE REVIEW

**Review by chemonique — APPROVED** [review]

**Review by Kenneth-Ngcobo — CHANGES_REQUESTED** [review]
Please remove dummy data, keep it local for testing.
details.css should be details.module.css

File: components/description/details.css
Comment (Renzo-codespace) [inline components/description/details.css:1]: Rather delete the file than committing and empty one.

File: components/instructions/instructions.js
Comment (Renzo-codespace) [inline components/instructions/instructions.js:1]: Please delete unused code rather than commenting it out.

**Review by Renzo-codespace — COMMENTED** [review]
Please update your branch!
Dummy data will have to be deleted as soon as the db connection is live.
The rest looks great so far!

**Review by chemonique — COMMENTED** [review]
delete the empty CSS files, remove dummy data

**Pull request closed without merging**